jwst/ami/mask_definition_ami.py

```diff
@@ -0,0 +1,127 @@
+#
+#  Module for defining mask geometry in pupil space
+#  Replace old mask_definitions by reading info in nrm_model into a similar object.
+#
+
+import numpy as np
+import math
+
+from .utils import rotate2dccw
+
+m = 1.0
+mm = 1.0e-3 * m
+um = 1.0e-6 * m
```

Comment:
suggest capitalizing these

Reply:
I just removed them since they're actually no longer used in that file